shelley/chain-and-ledger/formal-spec/delegation.tex

```diff
@@ -568,10 +568,11 @@ \subsection{Delegation Rules}
   \begin{equation}\label{eq:deleg-mir}
     \inference[Deleg-Mir]
     {
+      \dom{(\fun{moveRewards}~\var{c})} \subseteq \dom{stkCreds} \\
```

Comment:
I'm not sure we need this check. This may be true at the time we check the MIR cert, but there is a lot of time from this until the end of the epoch when the accounts will be credited, and so keys can be both added and removed in the mean time. And `applyRUpd` already handles the potential changes in the keys in the MIR cert and what is registered at the time the update is applied.